edg/parts/JlcCapacitor.py

```diff
@@ -54,9 +57,10 @@ def parse_row(row: PartsTableRow) -> Optional[Dict[PartsTableColumn, Any]]:
         footprint = cls.PACKAGE_FOOTPRINT_MAP[row[cls._PACKAGE_HEADER]]
         extracted_values = cls.parse(row[cls.DESCRIPTION_COL], CAPACITOR_MATCHES)
 
-        tempco = extracted_values['tempco'][1]
-        if tempco[0] not in ('X', 'C') or tempco[2] not in ('R', 'S', 'G'):
-          return None
+        # some JLC part have no tempco info, but still want use it
+        # tempco = extracted_values['tempco'][1]
+        # if tempco[0] not in ('X', 'C') or tempco[2] not in ('R', 'S', 'G'):
+        #   return None
```

Comment:
I don't think this filtering step should be removed. Tempco doesn't just affect temperature stability and tolerances, I believe it's also correlated with capacitance derating - where at DC load, the capacitor has less capacitance. Worse capacitors (eg, Y5V) typically have worse derating (and are less 'ideal') than better ones (eg, X5R), though the benefit is that they have higher capacitance.

A future PR might address this by adding a parameter `CeramicCapacitor.minimum_tempco`, which would default to `X5R` (following the principle of erring on the side of overkill rather than wrong) but would still allow the user (via refinements) to specify an alternative.